### PR TITLE
Configure Ether value with unit

### PR DESCRIPTION
**Pull request by nkuba**
Currently `MaxGasPrice` and `BalanceAlertThreshold` config properties are passed in `wei`. This can be insufficient for bigger values.

In this PR we are utilizing change introduced to `keep-common` (https://github.com/keep-network/keep-common/pull/60) that let us define a value in a config file with Ether unit, e.g. `500 Gwei` or `10.5 ether`.

Refs: https://github.com/keep-network/keep-core/issues/2176
Depends on: https://github.com/keep-network/keep-common/pull/60
Depends on:  https://github.com/keep-network/keep-core/pull/2180

### CODE REVIEW

File: configs/config.toml.SAMPLE [hunk 1]
```diff
@@ -41,7 +41,7 @@
 	# balance below which the client will start reporting errors in logs.
 	# The value should be passed in [wei].
```
Comment (pdyraga) [inline configs/config.toml.SAMPLE:42]: Let's tweak this file the same way as we did in `keep-core`:
- remove `# The value should be passed in [wei].` comment,
- add examples of how to pass other units.

File: go.sum [hunk 2]
```diff
@@ -332,10 +332,19 @@ github.com/keep-network/cli v1.20.0/go.mod h1:nzsst4JjU+rGE8Q5J839fYxectxWHpLhxK
...
+github.com/keep-network/keep-common v1.2.1-0.20201125164243-e39a6818bc8a/go.mod h1:emxogTbBdey7M3jOzfxZOdfn139kN2mI2b2wA6AHKKo=
 github.com/keep-network/keep-core v1.3.2-0.20201117170744-88a808971677 h1:rOdyRpHh8pj5TW9OOk0oB2WOAfARbzkWM6AACx4StKw=
 github.com/keep-network/keep-core v1.3.2-0.20201117170744-88a808971677/go.mod h1:SlbEjr3r6alzEB5raBFCT8z7RJjEy/2MQjBWuyL1jkg=
+github.com/keep-network/keep-core v1.3.2-0.20201124140656-400a4ffeebdb h1:OfHSpHaUJbLGgb3n+Fr1C2VhWtKprMwxZ+jau0CaS4M=
+github.com/keep-network/keep-core v1.3.2-0.20201124140656-400a4ffeebdb/go.mod h1:SlbEjr3r6alzEB5raBFCT8z7RJjEy/2MQjBWuyL1jkg=
+github.com/keep-network/keep-core v1.3.2-0.20201124143107-e8b2e446cd77 h1:FiyArFMfs0woyrO7/a/4t02EJt1MWvVnCcbnPh+P1Gs=
+github.com/keep-network/keep-core v1.3.2-0.20201124143107-e8b2e446cd77/go.mod h1:Pos+BxBGM7HekIUvzV+FSuFdrdAQyFMdVt5RSMEMGQg=
+github.com/keep-network/keep-core v1.3.2-0.20201125171123-4549ea2b23dd h1:vetLmc+MdPd2Xt54S3SmjxHf8Y9Fhu7cM6dc7v9t7fA=
+github.com/keep-network/keep-core v1.3.2-0.20201125171123-4549ea2b23dd/go.mod h1:rh1L5xjGLmlOqIBCoamthTnkjTqlG01lmX1KHtVJL9g=
```
Comment (pdyraga) [inline go.sum:347]: `go mod tidy` should let us get rid of the duplicates.

File: go.mod [hunk 3]
```diff
@@ -18,8 +18,8 @@ require (
 	github.com/gogo/protobuf v1.3.1
 	github.com/google/gofuzz v1.1.0
 	github.com/ipfs/go-log v1.0.4
-	github.com/keep-network/keep-common v1.2.1-0.20201117163745-878626866ba7
-	github.com/keep-network/keep-core v1.3.2-0.20201117170744-88a808971677
+	github.com/keep-network/keep-common v1.2.1-0.20201125164243-e39a6818bc8a
+	github.com/keep-network/keep-core v1.3.2-0.20201125171123-4549ea2b23dd
```
Comment (pdyraga) [inline go.mod:22]: Having https://github.com/keep-network/keep-core/pull/2180 merged now, let's switch this to some commit from `master`.

**Comment by pdyraga** [comment]
Tested locally and works well. Confirmed the change is backward-compatible and any number passed without quotes as `MaxGasPrice` or `BalanceAlertThreshold` is interpreted as a value in wei. Also, confirmed "ether", "wei", and "Gwei" units are properly interpreted for those values.

File: configs/config.toml.SAMPLE [hunk 4]
```diff
@@ -16,7 +16,8 @@
 	# willing to pay for the transaction to be mined. The offered transaction
 	# gas price can not be higher than the max gas price value. If the maximum
 	# allowed gas price is reached, no further resubmission attempts are
-	# performed.
+	# performed.A value can be provided in `wei`, `Gwei` or `ether`, e.g.
```
Comment (pdyraga) [inline configs/config.toml.SAMPLE:19]: Missing whitespace, but we can fix it later.

**Review by pdyraga — APPROVED** [review]

File: configs/config.toml.SAMPLE (hunk 4 above)
Comment (nkuba) [reply configs/config.toml.SAMPLE:19]: 🤦

**Pull request merged**